superset/charts/api.py

```diff
@@ -279,6 +256,53 @@ def ensure_thumbnails_enabled(self) -> Optional[Response]:
 
     allowed_rel_fields = {"owners", "created_by", "changed_by"}
 
+    @expose("/<id_or_uuid>", methods=["GET"])
```

Comment:
haven't dug in, but isn't there an existing method that we can modify to now support uuid while preserving the API, essentially doing a non-breaking change that supports uuid as an input, and adds new uuid fields in the output?